cli/src/commands/workspace/update_stale.rs

```diff
@@ -125,6 +125,16 @@ fn create_and_check_out_recovery_commit(
             vec![commit_id.clone()],
             commit.tree_id().clone(),
         )
+        .set_description(
+            "recovery commit from `jj workspace update-stale`
```

Comment:
I don't have a strong feeling, but it would be nice if the description looked unusual. User might push the recovery commit by mistake if it looked like a normal commit.

Reply:
Maybe we should prefix it with `JJ:` and use that private commits feature to ensure that commits starting with `JJ:` don’t get pushed by default.

Reply:
I went with just uppercasing for now since we don't interpret any prefix specially yet anyway.